public/components/ContentPanel/ContentPanel.tsx

```diff
@@ -22,15 +22,22 @@ import {
   EuiHorizontalRule,
   EuiPanel,
   EuiTitle,
+  EuiFormRow,
+  EuiText,
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
 } from '@elastic/eui';
 
 type ContentPanelProps = {
   title: string;
+  subTitle?: ReactElement;
```

Comment:
I changed it to React Node: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/development/public/components/ContentPanel/ContentPanel.tsx#L33

Maybe you can re-use it.

Reply:
reused